_posts/2023-11-20-testing-quarkus-with-citrus.adoc

```diff
@@ -0,0 +1,506 @@
+---
+layout: post
+title: 'Testing Quarkus with Citrus'
+date: 2023-11-20
+tags: testing event-driven integration kafka messaging
+synopsis: 'Explore how to verify Quarkus event-driven applications with the Citrus integration test framework.'
+author: christophd
+---
+:imagesdir: /assets/images/posts/citrus
+
+image::citrus-quarkus.png[Citrus & Quarkus,align="center"]
+
+This post shows how to combine Quarkus with the https://citrusframework.org[Citrus] test framework in order write automated tests for event-driven applications.
```

Comment:
Missing 'to' after 'order': 

This post shows how to combine Quarkus with the https://citrusframework.org[Citrus] test framework in order to write automated tests for event-driven applications.